controllers/user.server.controller.js

```diff
@@ -1,13 +1,32 @@
 const User = require('../models/user.server.model');
 
+/**
+ * Returns the hashPassword given a plaintextPassword
+ * @param {String} password
+ */
+ function hashPassword(password) {
+    // TODO: Create a has function for the password
+    return 'password'
+}
+
 /**
  * Creates a new forum user using HTTP request object data
  * @param req HTTP request object
  * @param res HTTP request response object
  */
 exports.userCreate = function(req, res) {
-    // TODO: implement userCreate()
-    res.json({ dummyTest: "userCreate() dummy test passes" });
+    const
+        username = req.body.username,
+        name = req.body.name,
+        email = req.body.email,
+        hashedPassword = hashPassword(req.body.password);
+
+    User.create(
+        username,
+        name,
+        email,
+        hashedPassword
+    );
```

Comment:
```suggestion
User.create(username, name, email, hashedPassword, function(result) {
        if (result.err) {
            // Return the error message with the error status
            res.status(result.status).send(result.err);
        } else {
            // Return the forum post document object with 201 status
            res.status(201).json({"forumPost": result});
        }
    });
```
I recommend something like this. You could also include validation of request body fields.